lib/linux_admin/security/common.rb

```diff
@@ -0,0 +1,14 @@
+module LinuxAdmin
+  class Security
+    module Common
+      def replace_config_line(new_line, rep_regex, file_text)
+        new_text = file_text.gsub!(rep_regex, new_line)
```

Comment:
Although I would still be returning the same string object in the case where there are no substitutions made ... That might be even more confusing.